src/pages/projects/Project.tsx

```diff
@@ -19,7 +19,7 @@ export const Project = ({ image, title, description, tags, href }: ProjectProps)
 
   return (
     <div className="mb-8 border-b pb-8 flex flex-col gap-y-4">
-      {image && <img src={image} alt={title} className="border-white w-full rounded-lg border-4" />}
+      {image && <img src={image} alt={title} className="border-white rounded-lg border-4 w-full min-w-img-md min-h-img-md aspect-ratio-16/9" />}
```

Comment:
The issue was not very well described, apologies. In terms of the Project images, I think we can simply things and just give the images some minimum height while loading by removing the class names added in this PR and adding:

```
w-full min-h-32
```